pkg/playground/pkgs.js

```diff
@@ -0,0 +1,41 @@
+var $ = require("jquery");
+var cockpit = require("cockpit");
+
+$(function() {
+    var proxy = cockpit.dbus(null, { bus: "internal" }).proxy("cockpit.Packages", "/packages");
+
+    var manifests;
+
+    function update(str) {
+        var new_m = JSON.parse(str), p;
+
+        if (manifests) {
+            for (p in new_m) {
+                if (!manifests[p])
+                    console.log("ADD", p);
```

Comment:
This is in the playground, and wont happen normally.  But since the playground is used during testing, I will comment them out.